src/operator/tensor/elemwise_binary_op_basic.cc

```diff
@@ -10,6 +10,16 @@ namespace mxnet {
 namespace op {
 MXNET_OPERATOR_REGISTER_BINARY(elemwise_add)
 .add_alias("_add").add_alias("_plus").add_alias("_Plus")
+.describe(R"code(Adds arguments element-wise.
+
+Example::
+
+  >>> x = mx.nd.array([1, 2, 3, 4])
```

Comment:
@indhub, python specific examples should go to ndarray_doc.py. Also this does not have to be specific to Python.